### PR TITLE
Add ESD Strategy

### PR DESCRIPTION
**Pull request by l3wi**
Changes proposed in this pull request:
- Adds `esd` strategy to read bonded ESD tokens
- Adds a `readme.md` for the strategy
- Adds an `example.json` file to test locally.

### CODE REVIEW

File: src/strategies/esd/example.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+                "token": "0x36F3FD68E7325a35EB768F1AedaAe9EA0689d723",
+                "decimals": 18
+            }
+        },
+        "network": "1",
+        "addresses": [
+            "0x5b3d22cf16963eb57fea96482728d91e885cb71e",
+            "0xd1991b6fd521a3f357d96a15956702ce33342fec"
+        ],
+        "snapshot": "latest"
```
Comment (bonustrack) [inline src/strategies/esd/example.json:19]: It's better to use a specific block number here, this way we will always get the same result.

File: src/strategies/esd/index.ts [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+  space,
+  network,
+  provider,
+  addresses,
+  options,
+  snapshot
+) {
+  const blockTag = typeof snapshot === 'number' ? snapshot : 'latest';
+  // Fetch DAO Bonded tokens
+  const daoBondedBalance = await multicall(
```
Comment (bonustrack) [inline src/strategies/esd/index.ts:47]: The 3 multicalls could be merged into a single one, if you can do this that would make the strategy resolve much faster.

File: src/strategies/esd/example.json (hunk 1 above)
Comment (l3wi) [reply src/strategies/esd/example.json]: Snapshot added

File: src/strategies/esd/index.ts (hunk 2 above)
Comment (l3wi) [reply src/strategies/esd/index.ts]: Merged

**Comment by l3wi** [comment]
Updated code to reflect requested changes and merged upstream changes

File: src/strategies/esd/index.ts [hunk 3]
```diff
@@ -0,0 +1,97 @@
+import { formatUnits } from '@ethersproject/units';
+import { multicall } from '../../utils';
+
+export const author = 'lewi';
```
Comment (bonustrack) [inline src/strategies/esd/index.ts:4]: This must be your GitHub username
```suggestion
export const author = 'l3wi';
```

**Review by bonustrack — APPROVED** [review]
Works well! Great work on the PR.

**Pull request merged**